test/CorefMetricTestConfig.pm

```diff
@@ -356,6 +347,45 @@ our @TestCases = (
                           #"ceafm" => ,
                           #"ceafe" => ,
                           "blanc" => [0.13333, 0.18182, 0.15385] }
+},
+{ id => "O1", 
+    key_file => "DataFiles/TC-O.key", 
+    response_file => "DataFiles/TC-O-1.response",
+    allow_multi_tag => 1,
+    expected_metrics => { "muc" => [1.0, 1.0, 1.0],
+                          "bcub" => [1.0, 1.0, 1.0],
+                          "ceafm" => [1.0, 1.0, 1.0],
+                          "ceafe" => [1.0, 1.0, 1.0],
+                          "blanc" => [1.0, 1.0, 1.0] }
+},
+{ id => "O2", 
+    key_file => "DataFiles/TC-O.key", 
+    response_file => "DataFiles/TC-O-2.response",
+    allow_multi_tag => 1,
+    expected_metrics => { "muc" => [3/4, 1.0, 0.857142857142857],
+                          "bcub" => [0.75, 1.0, 0.8571],
+                          "ceafm" => [0.857142857142857, 1.0, 0.923076923076923],
+                          "ceafe" => [0.952380952380952, 0.952380952380952, 0.952380952380952],
+                          "blanc" => [0.554945054945055, 0.818181818181818, 0.65530303030303] }
+},
+{ id => "O3", 
+  key_file => "DataFiles/TC-A.key", 
```

Comment:
historically we created a TC-<letter>.key key and various test responses were created as TC-<letter>-<number>.response. so that you can match the key to the various responses easily when checking the files.
in this case you are using a different test case key for this test. the danger is that if someone changes case A, then it will have a side-effect that might not be expected.
could you please create another key, TC-P.key and rename TC-O-3.response to TC-P-1.response.
ALSO, add these three cases to the test/TestCases.README file where we have documented all the other cases.